sql/dd/impl/sdi.cc

```diff
@@ -379,6 +379,7 @@ Sdi_type serialize(const Tablespace &tablespace) {
 bool CheckDefaultCompatibility(const RJ_Document &doc) {
   assert(doc.HasMember("mysqld_version_id"));
 
+#if !defined(XTRABACKUP)
```

Comment:
why do we need this ifndef? Don't we need this for SDI deserialize version check?